src/bin/reo.rs

```diff
@@ -278,6 +292,26 @@ pub fn main() -> Result<()> {
             let json = serde_json::to_string_pretty(&uni).unwrap();
             println!("Universe is: {}", json);
         }
+        Some(("link", subs)) => {
+            let args: Vec<&str> = subs.values_of("relfs").unwrap().collect();
+            let target = Path::new(args[0]);
+            let mut uni_target = Universe::load(target).unwrap();
```

Comment:
@Karina5005 as a general rule, it's better to avoid compound names of variables: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html